--- python-parsl.orig/parsl/tests/conftest.py
+++ python-parsl/parsl/tests/conftest.py
@@ -367,12 +367,12 @@
     return rep
 
 
-def pytest_ignore_collect(path):
-    if 'integration' in path.strpath:
+def pytest_ignore_collect(collection_path, config):
+    if 'integration' in str(collection_path):
         return True
-    elif 'manual_tests' in path.strpath:
+    elif 'manual_tests' in str(collection_path):
         return True
-    elif 'scaling_tests/test_scale' in path.strpath:
+    elif 'scaling_tests/test_scale' in str(collection_path):
         return True
     else:
         return False
